Jenkinsfile

```diff
@@ -0,0 +1,31 @@
+def runShellBuildStage(){
+    sh """
+        python3.11 -m venv .venv
```

Comment:
I think I mentioned this before - relying on the vagaries of path is not good.

Things like the python interpreter, venv, all of these things should be defined by variables with a fixed/absolute path

Reply:
Okay, will change, thank you.